docs/csharp/language-reference/builtin-types/value-tuples.md

```diff
@@ -75,46 +80,52 @@ C# supports assignment between tuple types that satisfy both of the following co
 
 Tuple element values are assigned following the order of tuple elements. The names of tuple fields are ignored and not assigned, as the following example shows:
 
-[!code-csharp-interactive[tuple assignment](snippets/shared/ValueTuples.cs#Assignment)]
+:::code language="csharp" interactive="try-dotnet-method" source="snippets/shared/ValueTuples.cs" id="Assignment":::
 
-You can also use the assignment operator `=` to *deconstruct* a tuple instance in separate variables. You can do that in one of the following ways:
+You can also use the assignment operator `=` to *deconstruct* a tuple instance in separate variables. You can do that in many ways:
 
 - Explicitly declare the type of each variable inside parentheses:
 
-  [!code-csharp-interactive[specify types of variables](snippets/shared/ValueTuples.cs#DeconstructExplicit)]
+  :::code language="csharp" interactive="try-dotnet-method" source="snippets/shared/ValueTuples.cs" id="DeconstructExplicit":::
 
 - Use the `var` keyword outside the parentheses to declare implicitly typed variables and let the compiler infer their types:
 
-  [!code-csharp-interactive[implicitly typed variables](snippets/shared/ValueTuples.cs#DeconstructVar)]
+  :::code language="csharp" interactive="try-dotnet-method" source="snippets/shared/ValueTuples.cs" id="DeconstructVar":::
 
 - Use existing variables:
 
-  [!code-csharp-interactive[existing variables](snippets/shared/ValueTuples.cs#DeconstructExisting)]
+  :::code language="csharp" interactive="try-dotnet-method" source="snippets/shared/ValueTuples.cs" id="DeconstructExisting":::
+
+Any of these can be combined in a single deconstruction expression.
```

Comment:
That's not true.
(1) and (2) cannot be combined, (2) and (3) cannot be combined, while (1) and (3) can be combined.
Currently, there is duplication in the docs between this section and the [Tuples section in the deconstruction article](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/deconstruct#tuples). The deconstruction doc is more complete, though. We should

- either copy content from the deconstruct article to this article
- or to remove duplication and reference one article from another
- something else

Reply:
I can create the separate related PR after we come to the conclusion and merge this PR, as this PR is really not about deconstruction.

Reply:
Thanks @pkulikov 

I'll fix this text so that it's correct. Over the medium term, I want to update the articles on discards, deconstruction, and tuples to more closely align with dotnet/csharpstandard#664, the updates in the standard. The main shift in focus is to consider discards a unique kind of local variable, and then consider deconstruction and tuples as ways to declare local variables.